### ISSUE TITLE
Setting Clarification

### ISSUE THREAD

**Issue by MySkypeLab** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#1966]
Can some additional information be supplied on this setting "To allow DSCP markings to be used for QoS, turn on Insert Quality of Service (QoS) markers for real-time media traffic." - The Admin center says "If you turn this on, all real-time media traffic for meetings will be marked." - This setting doesn't appear to affect media from clients so is it talking about meeting traffic generated from the cloud towards the client? (ie. for when ExpressRoute is used)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3c8aee4f-05cd-85bf-8e8a-e11fa62b4f08
* Version Independent ID: addd3d3c-e615-67e1-0678-c1d4dc451815
* Content: [Manage meeting settings](https://docs.microsoft.com/en-us/microsoftteams/meeting-settings-in-teams)
* Content Source: [Teams/meeting-settings-in-teams.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/meeting-settings-in-teams.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@officedocsbot assign @scanum

**Comment by lindspea** [comment]
@MySkypeLab Thank you for reaching out.  I'll get back to you soonest with an answer from the team.

**Comment by lindspea** [comment]
@LanaChin

**Comment by LanaChin** [comment]
I'll ping the PM again.  Thanks.

**Comment by scanum** [comment]
@LanaChin Any updates from the PM? Thanks!

**Comment by LanaChin** [comment]
I've pinged twice since. Let me try again.

**Comment by scanum** [comment]
@LanaChin Any updates? Thanks

**Comment by scanum** [comment]
@LanaChin Any updates? Thanks

**Comment by scanum** [comment]
@LanaChin Following up on this issue. Any updates? Thanks!

**Comment by scanum** [comment]
@LanaChin Following up on this issue. Please let me know if there are any updates. Thanks!

**Comment by LanaChin** [comment]
Closing as this issue was addressed in #2828. If there are still questions around this, can reopen. Thank you.
